### PR TITLE
Update contact email address

### PR DESCRIPTION
**Pull request by olblak**
I configured the updatecli.io domain so I can stop using my personnal domain for updatecli.

### CODE REVIEW

**Review by lemeurherve — APPROVED** [review]

**Review by lemeurherve — CHANGES_REQUESTED** [review]
contact instead of conduct?

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ representative at an online or offline event.
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be
 reported to the community leaders responsible for enforcement at
-me@olblak.com.
+conduct@updatecli.io.
```
Comment (lemeurherve) [inline CODE_OF_CONDUCT.md:63]:
```suggestion
contact@updatecli.io.
```

**Review by lemeurherve — CHANGES_REQUESTED** [review]

**Review by lemeurherve — DISMISSED** [review]
Nevermind, read too quickly.

**Comment by olblak** [comment]
I am fine with both, while initially thinking of using contact@updatecli.io
I realize that cncf use conduct@cncf.io as you can see here https://github.com/cncf/foundation/blob/main/code-of-conduct.md#reporting

**Comment by lemeurherve** [comment]
Is this mandatory? If not, I would simply use contact for now.

**Comment by olblak** [comment]
don't delete comment while I am answering.
I think you are right to only use one email address

**Comment by lemeurherve** [comment]
> don't delete comment while I am answering.
Was creating another comment with the content I've added to my first comment, sorry, bad habit ^^

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (olblak) [inline CODE_OF_CONDUCT.md:63]:
```suggestion
contact@updatecli.io.
```

**Comment by olblak** [comment]
@lemeurherve Since you requested a change, feel free to approve this PR :)

**Review by lemeurherve — APPROVED** [review]

**Pull request merged**